.github/CODEOWNERS

```diff
@@ -373,6 +373,8 @@ packages/react-components/react-motion-components-preview/library @microsoft/tea
 packages/react-components/react-motion-components-preview/stories @microsoft/teams-prg
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31976&commitId=43f73c49ed3ba8825a062a1f867aaa693dcd9543&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|3| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|2| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>